src/examples/defaultscene.js

```diff
@@ -1,40 +1,43 @@
 // default scene loaded in src/engine/engine.js
-
 import {
   Scene,
   TorusBufferGeometry,
-  MeshNormalMaterial,
+  DirectionalLight,
   Mesh,
   Vector3,
+  MeshStandardMaterial,
+  Color,
 } from "three";
 const scene = new Scene();
 
-scene.init = () => {
-  scene.traverse(e => {
-    scene.remove(e);
-  });
+const rings = [
+  [new Vector3(1, 0, 1), new Color(0xff0000), 0.2],
+  [new Vector3(1, -1, 0), new Color(0x00ff00), 0.15],
+  [new Vector3(0, 1, 0), new Color(0x0000ff), 0.1],
```

Comment:
What if you made this an object with properties like `{rotation: new Vector(), color: ..., scale: ...}`? Then in the foreach loop below you could refer to the property which might read a better.

Reply:
one of those cases where I toyed with 4 different data structures and missed the most straightforward one. Done, thanks!